website/www/site/content/en/case-studies/akvelon.md

```diff
@@ -17,3 +26,154 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- **Stateful processing with `DoFn` and timers**, which improves streaming reliability and reduces overall pipeline latency
+- **Full compatibility with Google Cloud Dataflow**, ensuring autoscaling, fault tolerance, and operational simplicity through managed Apache Beam execution
+
+This design provided both Protegrity and its enterprise clients with a reusable, open-source architecture for scalable data privacy and processing.
+
+### The Results
+- Enabled data tokenization at scale for regulated industries
+- Accelerated adoption of Dataflow templates across Protegrity’s customer base
+- Delivered an open-source Flex Template that benefits the entire Apache Beam community
+
+<blockquote class="case-study-quote-block case-study-quote-wrapped">
+  <p class="case-study-quote-text">
+    In collaboration with Akvelon, Protegrity utilized a Dataflow Flex template that helps us enable customers to tokenize and detokenize streaming and batch data from a fully managed Google Cloud Dataflow service. We appreciate Akvelon’s support as a trusted partner with Google Cloud expertise.
+  </p>
+  <div class="case-study-quote-author">
+    <div class="case-study-quote-author-img">
+        <img src="/images/case-study/akvelon/chitnis.png">
+    </div>
+    <div class="case-study-quote-author-info">
+        <div class="case-study-quote-author-name">
+          Jay Chitnis
+        </div>
+        <div class="case-study-quote-author-position">
+          VP of Partners and Business Development @Protegrity
+        </div>
+    </div>
+  </div>
+</blockquote>
+
+## Use Case 2: CDAP Connector for Apache Beam
+
+### The Challenge
+
+**CDAP** had extensive plugin support for Spark but lacked native compatibility with Apache Beam. This limitation prevented organizations from reusing CDAP's rich ecosystem of data connectors (e.g., Salesforce, HubSpot, ServiceNow) within Beam-based pipelines, constraining cross-platform integration.
+
+### The Solution
+
+Akvelon engineered a **shim layer** (CDAP Connector) that bridges CDAP plugins with Apache Beam. This innovation enables CDAP source and sink plugins to operate seamlessly within Beam pipelines.
+
+<div class="post-scheme">
+    <a href="/images/case-study/akvelon/diagram-02.png" target="_blank" title="Click to enlarge">
+        <img src="/images/case-study/akvelon/diagram-02.png" alt="CDAP Connector Integration with Apache Beam">
+    </a>
+</div>
+
+### Highlights
+
+- Supports `StructuredRecord` format conversion to Beam schema (`BeamRow`)
+- Enables CDAP plugins to run seamlessly in both Spark and Beam pipelines
+- Facilitates integration testing across third-party data sources (e.g., Salesforce, Zendesk)
+- Complies with Beam’s development and style guide for open-source contributions
+
+The project included prototyping, test infrastructure, and Salesforce plugin pipelines to ensure robustness.
+
+### The Results
+
+- Made **CDAP plugins reusable in Beam pipelines**
```

Comment:
for both cases, adding quantifiable metrics, if available (e.g., percentage increase or number of customers, data volume), will be much better.

Reply:
fixed